pkg/whip/whip_handler.go

```diff
@@ -94,22 +102,25 @@ func (h *whipHandler) Init(ctx context.Context, p *params.Params, sdpOffer strin
 
 	h.updateSettings()
 
+	offer := &webrtc.SessionDescription{
+		Type: webrtc.SDPTypeOffer,
+		SDP:  sdpOffer,
+	}
+	h.expectedTrackCount, err = h.validateOfferAndGetExpectedTrackCount(offer)
+	if err != nil {
+		return "", err
+	}
+
 	if p.BypassTranscoding {
 		h.sdkOutput, err = lksdk_output.NewLKSDKOutput(ctx, p)
 		if err != nil {
 			return "", err
 		}
+	} else if len(h.simulcastLayers) != 0 {
```

Comment:
I think that's a valid limitation to keep long term. If we really wanted to support this combination, we would need to only forward the HIGH quality layer to the transcoding pipeline.

Reply:
:+1: